src/Dapr.Actors/Runtime/ActorReminder.cs

```diff
@@ -72,5 +99,10 @@ public class ActorReminder : ActorReminderToken, IActorReminder
         /// Gets the reminder period.
         /// </summary>
         public TimeSpan Period { get; }
+
+        /// <summary>
+        /// Gets the number of invocations of the reminder left.
+        /// </summary>
+        public int RepetitionsLeft { get; }
```

Comment:
Do we need a default for people who call the original constructor?

Reply:
The default will be a zero which is fine.

Reply:
Can this property be renamed to match the parameter? `Repetitions`. I also would prefer that this reflect the behavior (1 instead of 1). I don't get why it has to be 0.